### ISSUE TITLE
The URL contains a regular expression.

### ISSUE THREAD

**Issue by wz2326** [TongchengOpenSource/smart-doc#81]
I encountered a problem:
URL path: `/detail/{id:[a-zA-Z0-9]{3}}/{name:[a-zA-Z0-9]{3}}`
The generated Postman interface is incorrect.

Here is my opinion:
```java
public static String formatAndRemove(String str, Map<String, String> values) {
    // /detail/{id:[a-zA-Z0-9]{3}}/{name:[a-zA-Z0-9]{3}}
    if (str.indexOf(":") >= 0) {
        String[] strArr = str.split("/");
        for (int i = 0; i < strArr.length; i++) {
            if (strArr[i].indexOf(":") >= 0) {
                String reg = strArr[i].substring(strArr[i].indexOf(":") + 1, strArr[i].length() - 1);
                Generex generex = new Generex(reg);
                // Generate random String
                String randomStr = generex.random();
                String key = strArr[i].substring(1, strArr[i].indexOf(":"));
                if (values.containsKey(key)) {
                    values.put(key, randomStr);
                }
                strArr[i] = strArr[i].substring(0, strArr[i].indexOf(":")) + "}";
            }
        }
        str = StringUtils.join(Arrays.asList(strArr), '/');
    }

    StringBuilder builder = new StringBuilder(str);
    Set<Map.Entry<String, String>> entries = values.entrySet();
    Iterator<Map.Entry<String, String>> iteratorMap = entries.iterator();
    while (iteratorMap.hasNext()) {
        Map.Entry<String, String> next = iteratorMap.next();
        int start;
        String pattern = "{" + next.getKey() + "}";
        String value = next.getValue().toString();
        // values.remove(next.getKey());
        // Replace every occurrence of {key} with value
        while ((start = builder.indexOf(pattern)) != -1) {
            builder.replace(start, start + pattern.length(), value);
            iteratorMap.remove();
            values.remove(next.getKey());
        }
    }
    return builder.toString();
}
```
Of course, I know you will have better ways.

**Comment by shalousun** [comment]
fixed in version 2.0.6

**Comment by wz2326** [comment]
Forgot to mention, there is a dependency in the code:
```xml
<dependency>
    <groupId>com.github.mifmif</groupId>
    <artifactId>generex</artifactId>
    <version>1.0.2</version>
</dependency>
```
